release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -503,6 +503,13 @@ With this release, you can view network information for ingress flows to {produc
 
 For more information, see  xref:../installing/install_config/configuring-firewall.adoc#network-flow-matrix_configuring-firewall[{product-title} network flow matrix].
 
+[id="ocp-release-notes-networking-metallb-dynamic-asn_{context}"]
+==== MetalLB updates for Border Gateway Protocol
+
+With this release, MetalLB includes a new field for the Border Gateway Protocol (BGP) peer custom resource. 
+You can use the `dynamicASN` field to detect the Autonomous System Number (ASN) to use for the remote end of a BGP session.  
+This is an alternative to explicitly setting an ASN in the `spec.peerASN` field.
```

Comment:
```suggestion
With this release, MetalLB includes a new field for the Border Gateway Protocol (BGP) peer custom resource. You can use the `dynamicASN` field to detect the Autonomous System Number (ASN) to use for the remote end of a BGP session. This is an alternative to explicitly setting an ASN in the `spec.peerASN` field.
```

- Avoid hard wrapping per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines

Reply:
Thanks for the review @xenolinux 
I don't think what I've done here is hard wrapping though. I did some Googling and read a bit about it [here](https://jesseduffield.com/Hard-Wrap-vs-Soft-Wrap/).  
What I've done is more like [Ventilated Prose](https://writetheasciidocs.netlify.app/ventilated-prose).

Reply:
ah okay. Thanks for the clarification.